### ISSUE TITLE
Import and export databases

### ISSUE THREAD

**Issue by losfair** [losfair/mvsqlite#5]
Import from and export to SQLite database files.

**Comment by tabeth** [comment]
The naïve approach here would be to dump the entire database into its sql statements, run sqlite (not mvsqlite), run saved statements, use native tool to get `db`. Importing would be similar, where `mvstore` or another utility could target an existing db, dump the commands, stream them back into `mvsqlite`. 

I'm not sure if other ways would go well with the limitations and guarantees FoundationDB exposes. Thoughts?

**Comment by fire** [comment]
I thought the standard ways of exporting sqlite worked. Can you clarify?

**Comment by losfair** [comment]
Yes the standard way works. Providing an API to dump on the page layer directly is more efficient though.
